app/routes/events/EventCreateRoute.js

```diff
@@ -55,8 +100,10 @@ const mapDispatchToProps = {
 
 export default compose(
   replaceUnlessLoggedIn(LoginPage),
+  prepare(({ location }, dispatch) => dispatch(fetchEvent(location.query.id))),
```

Comment:
What if this is a `create`, and `query.id` is `undefined`?

Reply:
Hmm, I thought about that when when returning initialValues, but forgot to check the dispatch. Good catch